packages/dataviews/src/style.scss

```diff
@@ -401,6 +401,7 @@
 	li {
 		margin: 0;
 		cursor: pointer;
+		border-top: 1px solid $gray-100;
```

Comment:
Move the border to the topmost HTML element. The `.dataviews-view-list__item` only targets the item but not the actions.